numba/tests/test_array_methods.py

```diff
@@ -30,6 +30,9 @@ def np_around_unary(val):
 def np_round_array(arr, decimals, out):
     np.round(arr, decimals, out)
 
+def np_round__array(arr, decimals, out):
```

Comment:
What is the double-underscore variant of this function and test for? It looks identical to the version above - am I missing something?

Reply:
There's `round` and `round_` in NumPy, so, I'm trying to continue the pattern.

Reply:
The function calls `np.round` though, rather than `np.round_`.

Reply:
Ooops.